polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -134,3 +158,14 @@ export function Item({
     </>
   );
 }
+
+const truncateText = (text: string) => {
+  const trimmedText = text.trim();
+  return (
+    <Tooltip content={trimmedText} zIndexOverride={514}>
+      <div className={styles.Trimmed}>
+        <Truncate>{trimmedText}</Truncate>
+      </div>
```

Comment:
```suggestion
      <Text truncate as="p" variant="bodySm">{trimmedText}</Text>
```

The undocumented `Truncate` component is going to be removed in favor of `Text.truncate`.